models/localized-string/src/generator.ts

```diff
@@ -0,0 +1,18 @@
+import type { TLocalizedString } from './types';
+
+import { Generator, fake } from '@commercetools-test-data/core';
+
+/**
+ * A localized string is a JSON object where the keys are of ↗ IETF language tag,
+ * and the values the corresponding strings used for that language.
+ */
```

Comment:
Wondering if the convention should be a JSDoc comment or just a comment pointing to the docs. We have both variants in mc test-data.

```suggestion
// https://docs.commercetools.com/api/types#localizedstring
```

Reply:
Good point, a link would be enough for me.